src/components/notification/notification.ts

```diff
@@ -38,6 +38,7 @@ const DEBOUNCE_TIME = 150;
  */
 @customElement('sbb-notification')
 export class SbbNotificationElement extends LitElement {
+  // FIXME could inherit from SbbOverlayBaseElement but readme generator is flawed
```

Comment:
Could we address this somehow?

Reply:
On our side, i think we have two ways:
- remove the abstract open/close methods from the `SbbOpenCloseBaseElement` class;
- change the visibility of the `open` method from private to public in the two involved components

Reply:
As discussed, the fixme is updated with the link to https://github.com/open-wc/custom-elements-manifest/issues/253